Include <ctime> header file in powertop to be able to build with
libc++. b/63890677.

diff -Nuar old/powertop-2.8/src/lib.h new/powertop-2.8/src/lib.h
--- old/powertop-2.8/src/lib.h
+++ new/powertop-2.8/src/lib.h
@@ -56,7 +56,7 @@
 
 
 
-
+#include <ctime>
 #include <string>
 using namespace std;
 
